docker/cygnus-ngsi/multisink_agent.conf

```diff
@@ -0,0 +1,400 @@
+cygnus-ngsi.sources.http-source-postgis.interceptors.nmi.type = com.telefonica.iot.cygnus.interceptors.NGSINameMappingsInterceptor$Builder
+cygnus-ngsi.sources.http-source-postgis.interceptors.nmi.name_mappings_conf_file = /opt/apache-flume/conf/name_mappings.conf
+
+# NGSIPostgisSink configuration
+cygnus-ngsi.sinks.postgis-sink.type = com.telefonica.iot.cygnus.sinks.NGSIPostgisSink
+cygnus-ngsi.sinks.postgis-sink.channel = postgis-channel
+#cygnus-ngsi.sinks.postgis-sink.enable_encoding = false
+#cygnus-ngsi.sinks.postgis-sink.enable_name_mappings = false
+#cygnus-ngsi.sinks.postgis-sink.enable_grouping = false
+#cygnus-ngsi.sinks.postgis-sink.enable_lowercase = false
+cygnus-ngsi.sinks.postgis-sink.postgis_host = iot-postgis
+cygnus-ngsi.sinks.postgis-sink.postgis_port = 5432
+cygnus-ngsi.sinks.postgis-sink.postgis_username =
+cygnus-ngsi.sinks.postgis-sink.postgis_password =
+#cygnus-ngsi.sinks.postgis-sink.attr_persistence = row
+#cygnus-ngsi.sinks.postgis-sink.attr_native_types = false
+#cygnus-ngsi.sinks.postgis-sink.batch_size = 1
+#cygnus-ngsi.sinks.postgis-sink.batch_timeout = 30
+#cygnus-ngsi.sinks.postgis-sink.batch_ttl = 10
+#cygnus-ngsi.sinks.postgis-sink.backend.enable_cache = false
+
+
+# ElasticSearch http-source
+
+cygnus-ngsi.sources.http-source-elasticsearch.type = org.apache.flume.source.http.HTTPSource
+cygnus-ngsi.sources.http-source-elasticsearch.channels = elasticsearch-channel
+cygnus-ngsi.sources.http-source-elasticsearch.port = 5058
+cygnus-ngsi.sources.http-source-elasticsearch.handler = com.telefonica.iot.cygnus.handlers.NGSIRestHandler
+cygnus-ngsi.sources.http-source-elasticsearch.handler.notification_target = /notify
+cygnus-ngsi.sources.http-source-elasticsearch.handler.default_service = default
+cygnus-ngsi.sources.http-source-elasticsearch.handler.default_service_path = /
+cygnus-ngsi.sources.http-source-elasticsearch.interceptors = ts nmi
+cygnus-ngsi.sources.http-source-elasticsearch.interceptors.ts.type = timestamp
+cygnus-ngsi.sources.http-source-elasticsearch.interceptors.gi.type = com.telefonica.iot.cygnus.interceptors.NGSIGroupingInterceptor$Builder
+cygnus-ngsi.sources.http-source-elasticsearch.interceptors.gi.grouping_rules_conf_file = /opt/apache-flume/conf/grouping_rules.conf
+cygnus-ngsi.sources.http-source-elasticsearch.interceptors.nmi.type = com.telefonica.iot.cygnus.interceptors.NGSINameMappingsInterceptor$Builder
+cygnus-ngsi.sources.http-source-elasticsearch.interceptors.nmi.name_mappings_conf_file = /opt/apache-flume/conf/name_mappings.conf
+
+# NGSIElasticsearchSink configuration
+cygnus-ngsi.sinks.elasticsearch-sink.type = com.telefonica.iot.cygnus.sinks.NGSIElasticsearchSink
+cygnus-ngsi.sinks.elasticsearch-sink.channel = elasticsearch-channel
+cygnus-ngsi.sinks.elasticsearch-sink.elasticsearch_host = localhost
+cygnus-ngsi.sinks.elasticsearch-sink.elasticsearch_port = 9200
+cygnus-ngsi.sinks.elasticsearch-sink.ssl = false
+#cygnus-ngsi.sinks.elasticsearch-sink.index_prefix = cygnus
```

Comment:
Are the commented lines correct in this file? (I think they are used as "templates" to be replaced by sed or something like that, but I'm not fully sure...)

Reply:
Yes, it's a template based on previous agent.conf: https://github.com/telefonicaid/fiware-cygnus/blob/master/docker/cygnus-ngsi/agent.conf

Reply:
Ok. NTC